### PR TITLE
Model-1: Logistic Regression & Decision Tree

### PR DESCRIPTION
**Pull request by niketan-byte**
This PR updates

- the dataset named as initial dataset, which has around 14k tweets
- the data preprocessing code which includes data reading, cleaning
- the logistic regression code for model-1, which includes the experiment recording using MarkovML
- the decision tree code for model-1, which includes the experiment recording using MarkovML

### CODE REVIEW

File: models/model1_part1.ipynb [hunk 1]
```diff
@@ -0,0 +1,567 @@
+{
```
Comment (gaursagar21) [inline models/model1_part1.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #2.    with zipfile.ZipFile("Datasets.zip","r") as zip_ref:
</pre><p>Any reason this is extracted before the imports?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PanwarJayant/spot/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (gaursagar21) [inline models/model1_part1.ipynb:1]: <p>This is a well written block.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PanwarJayant/spot/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (gaursagar21) [inline models/model1_part1.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    # removes HTML tags and removes punctuation from the text
</pre><p>Move this comment inside the function.</p><p> </p><p>Use block quote <code>"""</code> to enclose comments </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PanwarJayant/spot/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='6'/>

Comment (gaursagar21) [inline models/model1_part1.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    MODEL_NAME_2 = f"model_1_DT_try1_{int(time.time())}"
</pre><p>use better variable names. MODEL_NAME_2, recorder_2 isn't descriptive.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PanwarJayant/spot/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Comment (niketan-byte) [reply models/model1_part1.ipynb:1]: <p>There is no specific reason for that, it was just the error encountered while unzipping the zip file in the MarkovML notebook. The first basic !unzip command was not running, we had to use the import zipfile (the second way) for unzipping the file. May I now remove it?</p>

**Pull request merged**